streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -800,35 +800,39 @@ record = null;
                 record = null;
                 throw new TaskCorruptedException(Collections.singleton(id));
             }
+        } catch (final FailedProcessingException failedProcessingException){
```

Comment:
Checkstyle is complaining here. You need to change it to the following:
```suggestion
        } catch (final FailedProcessingException failedProcessingException) {
```

Reply:
done